tasks/build-burp.yml

```diff
@@ -83,9 +83,6 @@
 
 - name: Make install
   shell: cd {{ download_dir }}/{{ burpsrc }} && make install creates={{ burp_bin_path }}
-  notify: 
-    - restart burp server
-    - restart burp server systemd
```

Comment:
During upgrades, it is required 
@planet-winter

Reply:
> During upgrades, it is required
Yes!
However ansible could not find the given handlers as there is no handlers directory in the client role. If this is in the server role only, the client role only runs together with the server role. -> should add a handlers dir and handlers main.yml